src/Reducers/initialState.js

```diff
@@ -36,6 +36,9 @@ const initialState = {
     selectedAll: false,
     requestedFor: [],
     selectedOption: []
+  },
+  requestInboxAlert: {
```

Comment:
`RequestInboxAlert` it seems too specific to me. Cuz request have `inbox` `sent` and `archive`.  Maybe a little broader name